builds/misc/templates/build-packages.yaml

```diff
@@ -308,18 +308,17 @@ stages:
               set -ex
               git clone --recurse-submodules https://github.com/Azure/iot-identity-service.git
               pushd iot-identity-service
-              packageVersion=$(git tag | grep 1.3.[0-9]*$ | sort --version-sort -r | head -1)
               sudo docker run --rm \
                 -v "$(Build.SourcesDirectory)/iot-identity-service:/src" \
                 -e "ARCH=$PACKAGE_ARCH" \
                 -e "OS=$OS:$OSVERSION" \
-                -e "PACKAGE_VERSION=$packageVersion" \
+                -e "PACKAGE_VERSION=$VERSION" \
```

Comment:
Iotedge and Iot-identity-service do not require to have the same version. The best we can do here is probably parse the version from https://github.com/Azure/iot-identity-service/blob/main/.github/workflows/packages.yaml#L68. 
Note: The URI is required to be updated if the new branch be created. (i.e. main -> release/1.4).

Reply:
Ideally, we would be tagging our release commits so that we could just set `VERSION=$(git describe)`. However, given that we do not do this, the next safest alternative is probably to add a `version.txt` at the root directory of `iot-identity-service` as we do for `iotedge`. Parsing CI files for a version string seems fragile.

Reply:
We do tag our release artifacts for a given release branch -- `main` is not a release branch; hence, no tag.
I like the idea of version.txt for the iot-identity-service repo.